src/WebSdk/Publish/Targets/Microsoft.NET.Sdk.Publish.props

```diff
@@ -11,4 +11,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\Containers\build\Microsoft.NET.Build.Containers.props"
+          Condition="Exists('$(MSBuildThisFileDirectory)..\..\..\Containers\build\Microsoft.NET.Build.Containers.props')" />
```

Comment:
we need to add a condition here on `EnableSdkContainerSupport` being `true`. We also need to flip `EnableSdkContainerSupport` to `true` as part of this work :)